_posts/2018-06-14-virtual-mesh.md

```diff
@@ -0,0 +1,20 @@
+---
+layout: post
+title: Virtual Mesh Hack Night
+slug: virtual-mesh
+text: Join our virtual mesh from anywhere and play with distributed services.
+location: "#virtualmesh:tomesh.net"
+locationLink: https://chat.tomesh.net/#/room/#virtualmesh:tomesh.net
+date: 2018-06-14
+startTime: '19:30'
+endTime: '21:30'
+---
+
+Some Toronto Mesh members are connected through a virtual mesh network. This is a transitional phase where our mesh nodes are connected to one another using an internet connection instead of a physical line or radio. We are nonetheless able to test out distributed applications as if we were connected through a physical mesh.
+
+This month we are meeting virtually on [#virtualmesh:tomesh.net](https://chat.tomesh.net/#/room/#virtualmesh:tomesh.net) to do some hands-on hacking. We will:
+
+- Set up a [Secure Scuttlebutt](https://www.scuttlebutt.nz/) network
+- Run streaming tests over [IPFS](https://ipfs.io)
```

Comment:
Something came up for me and I will miss this virtual mesh. Will @darkdrgn2k and @ASoTNetworks be able to facilitate this? I am probably able to help via phone only.